src/modules/menu.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable prefer-template */
+/* eslint-disable no-useless-concat */
+/* eslint-disable prefer-const */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable linebreak-style */
+
+
+const menu = () => {
+  const content = document.getElementById('content');
+
+  const firstDiv = document.createElement('div');
+  firstDiv.className = 'container';
+
+  content.appendChild(firstDiv);
+
+  const row = document.createElement('div');
+  row.className = 'row mt-4';
+
+  firstDiv.appendChild(row);
+
+  const col = document.createElement('div');
+  const col2 = document.createElement('div');
+  const col3 = document.createElement('div');
+  const col4 = document.createElement('div');
+  const col5 = document.createElement('div');
+  const col6 = document.createElement('div');
+
+  row.appendChild(col);
+  row.appendChild(col2);
+  row.appendChild(col3);
+  row.appendChild(col4);
+  row.appendChild(col5);
+  row.appendChild(col6);
```

Comment:
Another alternative is [`append`](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append) instead of `appendChild`:

```js
element.append(child1, child1, child3, ...)
```

Reply:
Thank you 👍 It is done!!